functions/src/create-contract.ts

```diff
@@ -26,66 +27,42 @@ import {
   getFreeAnswerAnte,
   getNumericAnte,
   HOUSE_LIQUIDITY_PROVIDER_ID,
-  MINIMUM_ANTE,
 } from '../../common/antes'
 import { getNoneAnswer } from '../../common/answer'
 import { getNewContract } from '../../common/new-contract'
 import { NUMERIC_BUCKET_COUNT } from '../../common/numeric-constants'
 
-export const createContract = newEndpoint(['POST'], async (req, _res) => {
-  const [creator, _privateUser] = await lookupUser(await parseCredentials(req))
-  let {
-    question,
-    outcomeType,
-    description,
-    initialProb,
-    closeTime,
-    tags,
-    min,
-    max,
-    manaLimitPerUser,
-  } = req.body || {}
-
-  if (!question || typeof question != 'string')
-    throw new APIError(400, 'Missing or invalid question field')
-
-  question = question.slice(0, MAX_QUESTION_LENGTH)
-
-  if (typeof description !== 'string')
-    throw new APIError(400, 'Invalid description field')
-
-  description = description.slice(0, MAX_DESCRIPTION_LENGTH)
```

Comment:
I guess we're now rejecting everything instead of doing truncation. Reasonable choice.

Reply:
Yeah, this seems better for users than silently accepting their overlong things and then truncating them on our end. Especially since it's impossible to edit question text after your market is live.